db_test.go

```diff
@@ -1146,6 +1146,26 @@ func Test_getRecordFromKey(t *testing.T) {
 	}
 }
 
+func TestErrWhenBuildListIdx(t *testing.T) {
+	ts := []struct {
+		err     error
+		want    error
+		notwant error
+	}{
+		{
```

Comment:
Some questions were raised by the review dog. Could you help fix them? Thanks!

Reply:
has fixed, pls check again, thanks